prodng/system.yml

```diff
@@ -158,3 +158,7 @@
 - dep:
     name: i/init-system-helpers/init-system-helpers_1.22_all.deb
     repo: main
+- dep:
```

Comment:
Adding 4.9  version of listed c++ so as to not bring in gcc5 as a dependency, for now.